### ISSUE TITLE
Download table data from endpoint

### ISSUE THREAD

**Issue by hialvaro** [openblocks-dev/openblocks#82]
Is it possible to have some endpoint that runs a particular Database Query and returns the json result? I need to exctract the data form a table to use it in a backend script.

**Comment by lmx1117** [comment]
Hi! Do you mean to export the data as json file after running the query like this?
![image](https://user-images.githubusercontent.com/116335720/204505702-3f6b3ccf-8a9b-431a-8758-22dc49257308.png)

**Comment by hialvaro** [comment]
@lmx1117  Yes, but could this be done from an endpoint that is requested by a remote script?

**Comment by lmx1117** [comment]
> @lmx1117 Yes, but could this be done from an endpoint that is requested by a remote script?

Sorry about that recall, I was going to ask you for more specific usage scenarios, so you want this be done from an endpoint that is requested by a remote script, and the script may not be on OB right?
![image](https://user-images.githubusercontent.com/116335720/204509332-975eaa20-e848-42bc-a501-f2ac42559f5f.png)

**Comment by hialvaro** [comment]
Yes. For example a bash script that makes a "cURL" request to an endpoint and gets the query result in JSON, so I can then use such data.

**Comment by lmx1117** [comment]
> Yes. For example a bash script that makes a "cURL" request to an endpoint and gets the query result in JSON, so I can then use such data.

That’s a really cool idea🤩! We can’t support this for now, but we will consider providing an open api endpoint in the future~~

**Comment by hialvaro** [comment]
Would be awesome to have that option, would open a world of posibilites :)
